go.mod

```diff
@@ -4,15 +4,26 @@ go 1.12
 
 require (
 	github.com/Songmu/prompter v0.0.0-20181014095714-d227c68538bd
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
 	github.com/alioygur/godash v0.0.0-20160919141744-af6b3da41c5a
 	github.com/dave/jennifer v1.3.0
 	github.com/emicklei/proto v1.6.10
 	github.com/emicklei/proto-contrib v0.0.0-20190206213850-73879796f936
+	github.com/go-kit/kit v0.9.0 // indirect
```

Comment:
I do not think we want to add `go-kit` and all the other packages that where added here.

I think you need to revet this file.

Reply:
Got it, I fix those right way.